docs/howtos/cluster-resource-override.md

```diff
@@ -179,6 +120,8 @@ The jsonPatchOverrides field supports the following fields:
     * There is a list of reserved variables that will be replaced by the actual values:
       * `${MEMBER-CLUSTER-NAME}`:  this will be replaced by the name of the `memberCluster` that represents this cluster.
 
+##### Example: Override Labels
+    
 For example, to add a label to the `ClusterRole` named `secret-reader` on clusters with the label `env: prod`,

```

Comment:
I wonder if it's better to say "to set the labels of the `ClusterRole` named `secret-reader`..." because below example resets labels field. Or use `/metadata/labels/cluster-name` instead with a caveat about empty labels.